content/en/blog/2023/sig-release-spotlight/sig-release-spotlight.md

```diff
@@ -0,0 +1,91 @@
+
+For an ideal path, this is what we suggest:
+
+- Get yourself familiar with the code, including how features are managed, the release calendar, and the overall structure of the Release Team.
+- Join the Kubernetes community communication channels, such as [Slack](https://communityinviter.com/apps/kubernetes/community) (#sig-release), where we are particularly active. 
+- Join the [SIG Release weekly meetings](https://github.com/kubernetes/community/tree/master/sig-release#meetings)
+ which are open to all in the community. 
+ Participating in these meetings is a great way to learn about ongoing 
+ and future projects that you might find relevant for your skillset and interests.
+
+Remember, every experienced contributor was once in your shoes, and the community is often more than willing to guide and support newcomers. Don't hesitate to ask questions, engage in discussions, and take small steps to contribute.
+<figure>
+  <img src="/blog/2023/sig-release-spotlight/sig-release-meetings.png" alt="sig-release-questions">
+</figure>
+
+**5. What is the Release Shadow Program and how is it different from other shadow programs included in various other SIGs?**
+
+The Release Shadow Program offers a chance for interested individuals to shadow experienced members of the Release Team throughout a Kubernetes release cycle. This is a unique chance to see all the hard work that a Kubernetes release requires across sub-teams. A lot of people think that all we do is cut a release every three months, but that’s just the top of the iceberg.
+
+Our program typically aligns with a specific Kubernetes release cycle, which has a predictable timeline of approximately three months. While this program doesn’t involve writing new Kubernetes features, it still requires a high sense of responsibility since the Release Team is the last step between a new release and thousands of contributors, so it’s a great opportunity to learn a lot about modern software development cycles at an accelerated pace.
+
+**6. What are the qualifications that you generally look for in a person to volunteer as a release shadow/release lead for the next Kubernetes release?**
+
+While all the roles require some degree of technical ability, some require more hands-on experience with Go and familiarity with the Kubernetes API while others require people who are good at communicating technical content in a clear and concise way. It’s important to mention that we value enthusiasm and commitment over technical expertise from day 1. If you have the right attitude and show us that you enjoy working with Kubernetes and or/release engineering, even if it’s only through a personal project that you put together in your spare time, the team will make sure to guide you. Being a self-starter and not being afraid to ask questions can take you a long way in our team.
+
+**7. What will you suggest to someone who has got rejected from being a part of the Release Shadow Program several times?**
+
+Keep applying.
```

Comment:
```suggestion
Keep applying!
```

Reply:
I think the full stop (.) would work better than the exclamation mark.